.eslintrc

```diff
@@ -0,0 +1,113 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+
+  "rules": {
+    // Maybe enable?
+    "one-var": 0,
+    "vars-on-top": 0,
+    "eqeqeq": 0,
+    "yoda": [0, "never"],
+    "no-use-before-define": 0,
+    "no-mixed-requires": [0, false],
+
+    // Discuss?
+    "new-cap": 1,
+    "no-debugger": 1,
+    "no-fallthrough": 1,
+    "no-inner-declarations": [1, "functions"],
+    "no-loop-func": 1,
+    "no-multi-spaces": 1,
+    "no-native-reassign": 1,
+    "no-negated-in-lhs": 1,
+    "no-nested-ternary": 1,
+    "no-new-func": 1,
+    "no-proto": 1,
+    "object-curly-spacing": [1, "always"],
+    "quote-props": [1, "as-needed"],
+    "radix": 1,
+
+    // Done, but up for discussion
+    "space-after-keywords": [1, "always"],
+    "space-before-blocks": [1, "always"],
+    "space-before-function-paren": [1, "never"],
+
+    // Pretty standard
```

Comment:
Vast majority of these are recommendations by ESLint. They're all pretty safe but not attached to any of them.